### PR TITLE
[16.0][ADD] account_invoice_recipient_bank_currency

### PR DESCRIPTION
**Pull request by LauraCForgeFlow**
This module computes the 'Recipient Bank' field on invoices considering the move's company and currency. It's useful for managing bank details in different currencies.

### CODE REVIEW

File: account_invoice_recipient_bank_currency/models/account_move.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+                bank_ids = move.bank_partner_id.bank_ids.filtered(
+                    lambda bank: (
+                        not bank.company_id or bank.company_id == move.company_id
+                    )
+                    and (not bank.currency_id or bank.currency_id == move.currency_id)
+                )
+                bank_ids = bank_ids.sorted(key=lambda bank: bank.sequence)
+                move.partner_bank_id = bank_ids[0] if bank_ids else move.partner_bank_id
+            else:
+                move.partner_bank_id = move.bank_partner_id
```
Comment (JordiMForgeFlow) [inline account_invoice_recipient_bank_currency/models/account_move.py]: no need to do this as it is already taken care of in the super()

Comment (LauraCForgeFlow) [reply account_invoice_recipient_bank_currency/models/account_move.py]: Done! :)

**Review by JordiMForgeFlow — APPROVED** [review]
code review 👍🏼

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
Functional + code review

**Pull request merged**